Test/spec/simd/meta/simd_int_arith2.py

```diff
@@ -34,8 +36,19 @@ def lane_width(self):
         return int(self.LANE_TYPE.replace('i', '').split('x')[0])
 
     @property
-    def get_test_data_with_const(self):
-        """test const vs const and param vs const"""
+    def get_unary_test_data_1(self):
```

Comment:
Why the `_1` suffix on this name? If there is a conflict with `get_unary_test_data` below, perhaps this method could have a more descriptive name?

Reply:
These test data are both for testing 'const vs const' and 'different lanes go through different if-then clauses' for unary ops, I don't have a good suffix name for these, how about `get_unary_test_data_others`?

Reply:
Maybe `get_unary_complex_test_data` or `get_unary_combination_test_data`?

Reply:
Thanks @tlively, I am OK on `get_unary_complex_test_data`, since we have another combination tests, `get_unary_combination_test_data` will cause misleading.

Reply:
Done.